File: Baekjoon_Online_Judge/02_conditional/07_boj_2480.py
dice = [int(x) for x in input().split()]

if dice[0] == dice[1] == dice[2]:
    money = 10000+dice[0]*1000
elif dice[0] != dice[1] != dice[2] and dice[0] != dice[2]:
    money = 100*max(dice)
elif dice[0] == dice[2] != dice[1] or dice[0] == dice[1] != dice[2]:
    money = 1000+dice[0]*100
else:
    money = 1000+dice[1]*100
print(money)

File: Baekjoon_Online_Judge/02_conditional/07_boj_2480.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main(){
    int a, b, c, money;
    cin >> a >> b >> c;
    if(a == b and b == c)
        money = 10000+a*1000;
    else if(a != b and b != c and a != c)
        money = 100*max({a, b, c});
    else{
        if((a == b and a != c) or (a != b and a == c))
            money = 1000+a*100;
        else
            money = 1000+b*100;
    }
    cout << money;
}